modules/odometry_and_time.py

```diff
@@ -40,4 +43,11 @@ def __init__(self,
         self.odometry_data = odometry_data
         self.timestamp = timestamp
 
-# pylint: enable=too-few-public-methods
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        representation = (
+            f"{self.__class__}, time: {int(self.timestamp)}\n" + f"{self.odometry_data}"
+        )
+        return representation
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation